papaparse.js

```diff
@@ -1457,6 +1457,37 @@ License: MIT
 			if (!input)
 				return returnable();
 
+			// Rename headers if there are duplicates
+			if (config.header)
+			{
+				let firstLine = input.split(newline)[0]
+				let headers = firstLine.split(delim) 
+				let separator = '_';
+				let header_map = []; 
+				let header_count = {};
+
+				for (var i in headers){
+					var header = headers[i]
+					if (isFunction(config.transformHeader))
+						header = config.transformHeader(header, i);
+					var header_name = header
+					if (header_map.includes(header_name)){
+						if (!(header in header_count)){
```

Comment:
I think its simpler to write as:

count = header_count[header_name] || '1';
header_name = header+separator+count;
header_count[header_name] = count +1;

Removeing the if else lines.

Reply:
I struggled to make it work without an if statement. The problem being that it would still add the suffix at the end. The best I got was:

`let count = header_count[header] || 0
if (count > 0) header_name = header+separator+count;
header_count[header] = count+1`

Have committed this for you consideration if you prefer.